stories/exercises/05-customize-theme-tokens.mdx

```diff
@@ -0,0 +1,120 @@
+import { Meta, ColorPalette, ColorItem } from "@storybook/blocks";
+
+# 05. Customize with theme tokens
+
+Now that we have content on our landing page, we're going look at how we can make wide sweeping changes using USWDS Design Tokens and theme settings.
+
+You can view the available theme settings by visiting the [USWDS Settings Table](https://designsystem.digital.gov/documentation/settings/#general-settings-2).
+
+Lets start by updating some of our [theme color tokens](https://designsystem.digital.gov/documentation/settings/#color-settings-2). USWDS theme color tokens are designed to be useful and effective for small and large projects.
+
+1. Pick a color palette and copy SCSS variables.
+
+  <h2>Colors</h2>
+  <h3>Earth tones</h3>
```

Comment:
**issue**: We should use markdown here. Plus, we can use `<br />` for a little more spacing.

```suggestion
  <br />
  
  ## Colors
  
  <br />
  
  ### Earth tones
```

Reply:
Resolved in [b78561b](https://github.com/Bixal/storybook-rapid-prototyping/pull/13/commits/b78561b80cae38c00d9de5a1dd2e882c5416f184)